db/migrate/20240404182018_create_new_org_and_rep_tables.rb

```diff
@@ -0,0 +1,85 @@
+class CreateNewOrgAndRepTables < ActiveRecord::Migration[7.1]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    create_table :accredited_representatives, id: false do |t|
+      t.string :registration_number, null: false
+      t.string :poa_code
+      t.string :types, array: true
+      t.string :first_name
+      t.string :middle_initial
+      t.string :last_name
+      t.string :full_name
+      t.string :email
+      t.string :phone
+      t.string :address_type
+      t.string :address_line1
+      t.string :address_line2
+      t.string :address_line3
+      t.string :city
+      t.string :country_code_iso3
+      t.string :country_name
+      t.string :county_name
+      t.string :county_code
+      t.string :international_postal_code
+      t.string :province
+      t.string :state_code
+      t.string :zip_code
+      t.string :zip_suffix
+      t.jsonb :raw_address
+      t.float :lat
+      t.float :long
+      t.geography :location, limit: { srid: 4326, type: 'st_point', geographic: true }
+      t.timestamps
+    end
+    # rubocop:enable Metrics/MethodLength
+    safety_assured do
```

Comment:
Just a suggestion. You can simplify this by adding `id: false, primary_key: :registration_number` in the `create_table` migration.

Reply:
Cool, thanks